From b32faf215e39d76aa140a7f667d033aac5701af2 Mon Sep 17 00:00:00 2001
From: Yanjiang Jin <yanjiang.jin@windriver.com>
Date: Thu, 18 Sep 2014 12:11:09 +0800
Subject: [PATCH 199/202] export symbol irq_set_affinity

If we build Cavium octeon watchdog driver as module, we would get the below error:

ERROR: "irq_set_affinity" [drivers/watchdog/octeon-wdt.ko] undefined!

So export this symbol.

Signed-off-by: Yanjiang Jin <yanjiang.jin@windriver.com>
Signed-off-by: Bin Jiang <bin.jiang@windriver.com>
---
 kernel/irq/manage.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/kernel/irq/manage.c b/kernel/irq/manage.c
index a79d267..6829eba 100644
--- a/kernel/irq/manage.c
+++ b/kernel/irq/manage.c
@@ -202,6 +202,7 @@ int __irq_set_affinity(unsigned int irq, const struct cpumask *mask, bool force)
 	raw_spin_unlock_irqrestore(&desc->lock, flags);
 	return ret;
 }
+EXPORT_SYMBOL_GPL(__irq_set_affinity);
 
 int irq_set_affinity_hint(unsigned int irq, const struct cpumask *m)
 {
-- 
1.8.2.1

